test/FisherPrice_Alpaka/CMakeLists.txt

```diff
@@ -2,12 +2,12 @@
 # SPDX-License-Identifier: Apache-2.0
 
 #We must specify that we want to use alpaka
-find_package(alpaka)
+find_package(alpaka QUIET)
 
 if(NOT alpaka_FOUND)
   message(STATUS "No alpaka install found, disabling build of FisherPrice_Alpaka")
```

Comment:
I would profit and replace the message with one more generic, given that likely we're going to build more than FisherPrice_Alpaka. Maybe: "Alpaka installation not found: support disabled."? But this can be done as well later.

Reply:
I think that can be left to later, the message is clear for this context. Any later general support for Alpaka would result in the `find_package` call being moved anyway.